### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Customize New Project_Color Contrast: The color contrast ratio for the selected text (white text on blue background) is 4.39:1 

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3325]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.4.3; #Color Contrast; #C+E-H; #Java tooling for IntelliJ/Eclipse;

**Environment Details:**
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

**OS Details: ​**
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.4.3

**Note: Same issue is observed for all the contents with white text on blue background**

**Repro Steps: ​**
1) Open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Search for 'Azure Toolkit for IntelliJ' in the Marketplace and press enter.
5) Tab till 'Install' button and press enter
6) Restart IntelliJ IDEA application, and tab till Tools-->Azure-->Azure Sign in
7) Sign in with any Azure subscription
8) Tab till File menu and click new-->Project
9) In the New Project dialog box, select Maven and press enter
10) Check "Create from archetype" check box and select "maven-archetype-webapp" and tab till Next and press enter
11) Specify the GroupId and ArtifactId and press Next button
12) Tab till "Maven home directory" combo-box and expand it
13) Check the color contrast ratio for the selected list item

**Actual Result:**
The color contrast ratio for the selected text (white text on blue background) is 4.39:1 which is < 4.5:1

**Expected Result:**
The color contrast should be >=4.5:1

**User Impact:**
Users with visual impairments will find it difficult to access the content

**MAS Reference:**
[MAS 1.4.3](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

![Bug_4_IntelliJ_Create WebApp_CCR](https://user-images.githubusercontent.com/41108985/62204528-f6ec3c00-b3aa-11e9-909a-508878efe85f.png)
![Bug_4_IntelliJ_Customize_CCR](https://user-images.githubusercontent.com/41108985/62204530-f6ec3c00-b3aa-11e9-8e3c-3c1ccb1a3470.png)
![Bug_4_IntelliJ_Deploy to Azure_CCR](https://user-images.githubusercontent.com/41108985/62204532-f784d280-b3aa-11e9-830e-70a09cb7bd1f.png)

#A11yTCS

**Comment by Flanker32** [comment]
The color of selected text are affected by IntelliJ theme, which is designed by IntelliJ and out of the scope of Toolkits. Besides, IntelliJ provide three theme by default and one of them is High contrast, which is designed for low vision users by high color contrast ratio, here is the screen shots.

![image](https://user-images.githubusercontent.com/12445236/64153341-3e119500-ce61-11e9-876b-be2c7b7ff7a5.png)


![image](https://user-images.githubusercontent.com/12445236/64153206-f25eeb80-ce60-11e9-81ef-31fad218a281.png)

![image](https://user-images.githubusercontent.com/12445236/64153267-14586e00-ce61-11e9-9b60-fa4b25476f3b.png)

**Comment by Flanker32** [comment]
Importance of experience: Medium, users with visual impairments may can not see complete information if color contrast ratio is not proper.
Quality of experience: not impacted. IntelliJ provide alter theme for users including the High contrast one with high color contrast ratio.
Alternate solutions: No alternate technologies or options can be leveraged.
Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@soumyatippapuram Please have a review.

**Comment by soumyatippapuram** [comment]
@Flanker32 Please provide the link to your AAT. This is required to follow Exclusion process. For more details, please refer : https://microsoft.sharepoint.com/teams/ceacc/SitePages/External%20Bug%20Process.aspx

**Comment by Flanker32** [comment]
Linked in [AAT](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @soumyatippapuram please help us review it, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to IntelliJ IDEA (https://youtrack.jetbrains.com/issue/IDEA-222108)

**Comment by soumyatippapuram** [comment]
#MAS1.4.3

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749757
